tpm2_pytss/ESAPI.py

```diff
@@ -245,7 +245,7 @@ def tr_close(self, esys_handle: ESYS_TR):
         esys_tr_ptr[0] = esys_handle
         _chkrc(lib.Esys_TR_Close(self._ctx, esys_tr_ptr))
 
-    def set_auth(
+    def tr_set_auth(
         self, esys_handle: ESYS_TR, auth_value: Union[TPM2B_AUTH, bytes, str, None]
     ):
         """Set the authorization value of an ESYS_TR.
```

Comment:
The docstring has a reference to set_auth, "Therefore set_auth needs to be called again after" which probably should be updated